### ISSUE TITLE
Notation issue: yi can mean either model output or training set value

### ISSUE THREAD

**Issue by johnholman** [udlbook/udlbook#257]
I've found myself very confused by the fact that yi (y in bold) can mean either the output of a supervised learning model when the input is xi, or the second element of a training set example whose first element is xi.

Sometimes yi can appear initially to have different meanings even within the same sentence, e.g.

The loss encourages each training output yi to have a high probability under the distribution Pr(yi|xi) computed from the corresponding input xi (figure 5.1).

Initially I read this sentence as a statement about the model's output for the ith training example, and could make no sense of it. Eventually it made sense when I realised that in this particular context the output of the model is not in fact y but a distribution over y and that the first yi refers to the second value in a training set example, not to the model output for that example.

Generally I'd suggest using a different letter for model output and training set target/label (e.g. t) and to avoid the term "output" for the training set element. Unlike xi (which does serve as an input) the yi part of a training instance is never itself an output (only something the model output should be close to)

For this particular sentence, maybe something like this would be less likely to confuse:

The loss encourages each target value ti in a training set example to be given a high probability under the distribution Pr(yi|xi) output by the model.

(Assuming this is actually what was meant!)

If changing all references to label/target as y to a different symbol isn't feasible at this stage, avoiding "output" in the context of a training example might help, e.g.

The loss encourages each yi target value in a training set example to be given a high probability under the distribution Pr(yi|xi) output by the model.

or maybe better and avoiding the "target" or "label" word if you don't like those:

The loss enourages the model to output a distribution Pr(yi|xi) that gives a high probability to the yi value seen in a training example with that xi.

Though I still think the two different meanings of yi have potential for confusion generally.
